cli/src/cluster_query.rs

```diff
@@ -248,12 +248,12 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .help("Wait interval seconds between submitting the next transaction"),
                 )
                 .arg(
-                    Arg::with_name("count")
-                        .short("c")
-                        .long("count")
```

Comment:
Please let's not break users like this. Probably `count` could be an alias of `num_batches`, if you don't want to handle the parameters separately.

Reply:
don't worry, I don't think I'm checking this in :)

Reply:
I cherry-picked this change into local branch, ready to run 'batch' test if you desire @carllin